manim/mobject/coordinate_systems.py

```diff
@@ -1084,6 +1085,69 @@ def construct(self):
             **kwargs,
         ).set_opacity(opacity=opacity)
 
+    def get_area(
+        self,
+        graph: "ParametricFunction",
+        x_range: Optional[Sequence[float]] = None,
```

Comment:
**Issue** Seeing l.1138 it should not be optional and should be `Tuple[float, float`

Reply:
```suggestion
        x_range: Optional[Tuple[float, float]] = None,
```

Reply:
Oops sorry yes I only fixed the `Optional` part, didn't the the `Tuple` part!